### DISCUSSION TITLE
Missing "./react/shallow" specifier in "zustand" package

### DISCUSSION THREAD

**Discussion by mmyang** [pmndrs/zustand#2118] [Bug report] [answered]
I'm trying to use useShallow in accordance with the example at (https://github.com/pmndrs/zustand/blob/main/docs/guides/prevent-rerenders-with-use-shallow.md#example) and I'm getting the following error: `[vite] Internal server error: Missing "./react/shallow" specifier in "zustand" package`

I'm using zustand version 4.4.3, which picks up the change that modified the shallow.ts files (https://github.com/pmndrs/zustand/commit/e414f7ccf41eae09517ee2dcb44c9f5ae8a35a25#diff-36a5ae288be6cd24f699f85c6958ee1f67012811aed096320417731fe8679ff0)

Is this a bug in Zustand or is my own project misconfigured?

**Comment by dai-shi** [comment, marked as answer]
Here's the entry:
https://github.com/pmndrs/zustand/blob/90915ad1b853bfaf3091e86f650b3b3d3be31562/package.json#L77-L85

Is there any mis-spelling?

I would try clean installing node_modules.

**Reply by mmyang** [reply]
Clean installing my node_modules and restarting server fixed it.  Sorry for the false report.

**Comment by sergeyampo** [comment]
I don't consider removing node_modules as a sustainable solution, is there any fixes for this?

**Reply by dbritto-dev** [reply]
@sergeyampo version?
